ui/src/widgets/ui-notification/UINotification.vue

```diff
@@ -62,8 +62,12 @@ export default {
         value: function () {
             // Get the value (i.e. the notification text content) from the last input msg
             const value = this.messages[this.id]?.payload
-            // Sanetize the html to avoid XSS attacks
-            return DOMPurify.sanitize(value)
+
+            // Sanetize the html to avoid XSS attacks.
+            // Allow 'script' tags to allow styling of the notification content.
+            // The FORCE_BODY is required to avoid 'style' tags (at the start of the value string) still being skipped.
```

Comment:
The comments here do not quite align to the code changes? The `Allow 'script' tags`, suggests you've called them out, but you've actually called out `style` tags in the code change?